examples/proc_raw.py

```diff
@@ -114,4 +114,5 @@ def image_or_plot(d):
         image_or_plot(forplot)
         d = select_pathway(d, d.get_prop("coherence_pathway"))
         fl.next("with coherence pathway selected")
+        plt.gca().set_title("select " + str(d.get_prop("coherence_pathway")))
```

Comment:
I added this to be more informative

Reply:
note that this is the wrong pathway for the IR -- Δp_1 is the first pulse, which inverts the population (magnetization along z) so it should be 0 (population has a coherence order of 0).  Δp_2 is the 90 pulse, which is followed by an echo, so it's +1 (echo converts to -1)

I do not have time right now to track if this is set in the data or in the preproc.  In either case, hacking the preproc function so that it gets set to the correct value is probably the way to go, though if it's set in the ppg, should write a small PR to just change the value set there and increment the version number.

Reply:
(please pull in @guinneak198 for help on that as needed!)

Reply:
Do I still need to do this? it looks like you made the requested change in load_data ("hack_..._IR_v3")